nomad/grapl-core.nomad

```diff
@@ -415,179 +433,217 @@ job "grapl-core" {
   group "grapl-graph-merger" {
     count = var.num_graph_mergers
 
-    network {
-      mode = "bridge"
-    }
+    //    network {
+    //      mode = "bridge"
+    //    }
 
     task "graph-merger" {
       driver = "docker"
 
       config {
-        image = "${var.container_registry}/grapl/graph-merger:${var.graph_merger_tag}"
+        image        = "${var.container_registry}/grapl/graph-merger:${var.graph_merger_tag}"
+        network_mode = "grapl-network"
       }
 
       env {
-        RUST_LOG           = "${var.rust_log}"
-        REDIS_ENDPOINT     = "${var.redis_endpoint}"
-        MG_ALPHAS          = "${local.alpha_grpc_connect_str}"
-        GRAPL_SCHEMA_TABLE = "${var.schema_table_name}"
-        AWS_REGION         = "${var.aws_region}"
+        GRAPL_AWS_ENDPOINT          = var.aws_endpoint
+        GRAPL_AWS_ACCESS_KEY_ID     = var.aws_access_key_id
+        GRAPL_AWS_ACCESS_KEY_SECRET = var.aws_access_key_secret
+        AWS_DEFAULT_REGION          = var.aws_region # boto3 prefers this one
+        AWS_REGION                  = var.aws_region
+        RUST_LOG                    = var.rust_log
+        RUST_BACKTRACE              = 1
+        REDIS_ENDPOINT              = var.redis_endpoint
+        MG_ALPHAS                   = local.alpha_grpc_connect_str
+        GRAPL_SCHEMA_TABLE          = var.schema_table_name
         # https://github.com/grapl-security/grapl/blob/18b229e824fae99fa2d600750dd3b17387611ef4/pulumi/grapl/__main__.py#L165
-        DEST_BUCKET_NAME      = "${var.subgraphs_merged_bucket}"
-        SOURCE_QUEUE_URL      = "${var.graph_merger_queue}"
-        DEAD_LETTER_QUEUE_URL = "${var.graph_merger_dead_letter_queue}"
+        DEST_BUCKET_NAME      = var.subgraphs_merged_bucket
+        SOURCE_QUEUE_URL      = var.graph_merger_queue
+        DEAD_LETTER_QUEUE_URL = var.graph_merger_dead_letter_queue
```

Comment:
This is incorrect; the value should be the retry queue.

See https://github.com/grapl-security/grapl/pull/1092 for further background.